diff --git a/config.def.h b/config.def.h
index adc7752..192ed28 100644
--- a/config.def.h
+++ b/config.def.h
@@ -18,8 +18,8 @@ static const char col_gray4[]       = "#eeeeee";
 static const char col_cyan[]        = "#005577";
 static const char *colors[][3]      = {
 	/*               fg         bg         border   */
-	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
-	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
+        [SchemeNorm] = { "#bdc1c6", "#011627", "#1d3b53" },
+        [SchemeSel]  = { "#080808", "#c792ea", "#e3d18a" },
 };
 
 /* tagging */
@@ -61,7 +61,7 @@ static const Layout layouts[] = {
 
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
+static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", "#011627", "-nf", col_gray3, "-sb", "#c792ea", "-sf", col_gray1, NULL };
 static const char *termcmd[]  = { "kitty", NULL };
 
 static const Key keys[] = {
